### ISSUE TITLE
The XML reference page is not prominent enough

### ISSUE THREAD

**Issue by MakisH** [precice/precice.github.io#318]
Feedback by @gdenayer:

Our [XML Reference](https://precice.org/configuration-xml-reference.html) page is not prominent enough. In fact, I also realized recently that I was always searching for it.

Suggestions:

- Let's rename it to `XML configuration reference`, to hit the keyword
- Let's link to it from each specific feature documentation page

By the way: The v3 configuration, which removes the additional layer of `<solver-interface>` (since it is now just `<precice-configuration>`), would lead to an easier to navigate table of contents.

**Comment by fsimonis** [comment]
Why not simply name it `Reference`?
It being XML is really only a technicality, and it is in the Configuration category.

![image](https://github.com/precice/precice.github.io/assets/13552216/a3e13ce8-0941-44df-94ce-c89ab133dce3)


From a structural perspective, the "Configuration" layer doesn't exist though.
Currently, Google shows the Page as such:

![image](https://github.com/precice/precice.github.io/assets/13552216/9d70ff03-22a6-4688-a790-3ec09d50c227)

Choosing the name `Reference` could be too generic in this case. #342 would solve this in the long run.

![image](https://github.com/precice/precice.github.io/assets/13552216/cd7c526d-900d-4002-a868-4b02f7495fd0)

**Comment by MakisH** [comment]
> Why not simply name it `Reference`? It being XML is really only a technicality, and it is in the Configuration category.

Good point about XML only being a technicality, that's why "XML reference" does not say much. That's why I suggested "XML **configuration** reference". This makes it easy to search for the user, but also keeps the technical specificity.

The "Reference" itself does not really say much to me. It could mean anything.

By the way, we always need to keep in mind that users understand "preCICE" sometimes as the core library, sometimes as a whole package. We still need to be specific of what we mean here, and allow the user to easily discover the configuration of other components.

**Comment by fsimonis** [comment]
> "XML configuration reference"

If someone clicks through the website then they go `preCICE.org > Docs > Configuration > XML Configuration Reference`. In this case `XML` is irrelevant and `Configuration` is redundant.

If someone searches for `XML`, then the search needs to index this. Title aside, it indexes the summary. So we need to add it there.

If someone comes from an external search engine, then we need to provide a useful site title and description. Breadcrumbs should help to display the context `Docs > Configuration`. There the description becomes more important.

> By the way, we always need to keep in mind that users understand "preCICE" sometimes as the core library, sometimes as a whole package.

So we need an effective way to redirect people that look for adapter configurations, instead of preCICE configurations. Maybe a mention in the summary could help and a warning at the start of the reference.

Another point why I would get rid of XML in the title is to prevent people to get misdirected. Let's say some official adapter at some point also uses XML as a configuration language, then there are two matches that are both XML configurations.

I currently don't see a good solution here for the title.
We definitely need to update the summary though. It currently reads:

>  On this page you find the complete configuration references of preCICE API.

**Comment by MakisH** [comment]
Added more links via https://github.com/precice/precice.github.io/pull/675

Closing this issue for now, reopen if there is a clear idea for a better name. Related: https://github.com/precice/precice.github.io/pull/675#issuecomment-3777761789
